xcode/Subconscious/Shared/Services/Noosphere/SphereFS.swift

```diff
@@ -45,6 +45,10 @@ public protocol SphereProtocol {
     func changes(_ since: String?) throws -> [String]
 }
 
+protocol SphereIdentityProtocol {
+    func identity() throws -> String
+}
```

Comment:
Allows exposing the minimal API surface to the Address Book view, which in turn makes it simpler to support Swift UI previews etc.

Reply:
Love it.